src/state/GlobalState.tsx

```diff
@@ -100,7 +101,7 @@ export class GlobalState extends PureComponent<Props> {
     return games
   }
   state: StateProps = {
-    category: storage.getItem('category') || 'epic',
+    category: (storage.getItem('category') as Category) || 'epic',
```

Comment:
Shouldn't this be
```suggestion
    category: (storage.getItem('category') as Category) || 'legendary',
```
?

Reply:
yes, noticed afterwards. Noticed that if you were using heroic and it was epic it was showing nothing as well. Sent the fix on latest commits